### PR TITLE
'ha-ma-adjust-cloudconfig-render-function'

### PR DESCRIPTION
**Pull request by calvix**
## Checklist

- [ ] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Comment by calvix** [comment]
ping @xh3b4sd

File: service/controller/key/control_plane.go [hunk 1]
```diff
@@ -70,6 +70,10 @@ func ControlPlaneLaunchTemplateName(getter LabelsGetter, masterID int) string {
 	return fmt.Sprintf("%s-master%d-launch-template", ClusterID(getter), masterID)
 }
 
+func ControlPlaneMasterIgnitionPath(path string, masterID int) string {
+	return fmt.Sprintf("%s%d", path, masterID)
+}
```
Comment (xh3b4sd) [inline service/controller/key/control_plane.go:75]: @calvix I am not sure about this in https://github.com/giantswarm/aws-operator/pull/2368. Why does that have to be separate? Should I add it to the other PR? Should I? Should we drop this?

**Comment by calvix** [comment]
I think we can close this as you incorporated your changes into the PR as well right?

**Comment by xh3b4sd** [comment]
We can close this. There is just the question above that needs answering in the diff.

File: service/controller/key/control_plane.go (hunk 1 above)
Comment (calvix) [reply service/controller/key/control_plane.go:75]: no, you sorted the ignition paths different way, this is not needed

**Pull request closed without merging**